examples/contrib/sops/README.md

```diff
@@ -24,23 +26,47 @@ or if your keys are already in `gpg`, it's possibe to run:
 kpt fn run --env SOPS_IMPORT_PGP="$(gpg --armor --export-secret-keys)" <folder>
 ```
 
+For AGE case this function introduces and additional `SOPS_IMPORT_AGE` ENV var
+that must contain the [SOPS age keys.txt file](https://github.com/mozilla/sops/blob/master/age/keys.txt).
+This file is needed for decryption. For encryption it's possible to use
+`SOPS_AGE_RECIPIENTS` ENV variable or parameter `age`.
+
+```sh
+kpt fn run --env SOPS_IMPORT_AGE="$(cat <file with age keys>.txt)" <folder>
+```
+
+of if SOPS is already working , it's possible to run:
+
+```sh
+kpt fn run --env SOPS_IMPORT_AGE="$(cat ~/.config/sops/age/keys.txt)" <folder>
+```
+
 ## Function invocation
 
 Get this example and try it out by running the following commands:
 
 ```sh
 # download sops kpt-function example
 kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/contrib/sops .
-# copy example key from sops project
-curl -fsSL -o sops/key.asc https://raw.githubusercontent.com/mozilla/sops/master/pgp/sops_functional_tests_key.asc
-# run the function
-kpt fn run --env SOPS_IMPORT_PGP="$(cat sops/key.asc)" sops/local-configs
+cd kpt-functions-catalog/examples/contrib/sops
```

Comment:
After you run `kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/contrib/sops .`, we will have directory `sops` locally.
So I guess we should run `cd sops` here.